CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ CHANGELOG
 - Fix an issue where emojis are printed in non-interactive mode. (fixes
   [#2871](https://github.com/pulumi/pulumi/issues/2871))
 
+- Promises/Outputs can now be directly exported as the top-level (i.e. not-named) output of a Stack.
+  (fixes [#2910](https://github.com/pulumi/pulumi/issues/2910))
```

Comment:
You aren't the one to introduce this, but I don't think these extra newlines are intentional/desired in the changelog.

Reply:
will fix :)